pkg/env/central_diagnostics.go

```diff
@@ -0,0 +1,6 @@
+package env
+
+var (
+	// DisableCentralDiagnostics is set to true to signal that diagnostic bundles or dumps should not contain any information about central or the environment that central runs in.
+	DisableCentralDiagnostics = RegisterBooleanSetting("ROX_DISABLE_CENTRAL_DIAGNOSTICS", false)
```

Comment:
Can we avoid the double negation here, instead using `ROX_ENABLE_CENTRAL_DIAGNOSTICS`?

Reply:
I decided against it because by default diagnostics always includes central env. This is turning it off, because that's non-default. Also I copied from `ROX_DISABLE_AUTOGENERATED_REGISTRIES` :p 

I don't feel too strongly though so I can change it if you do feel strongly on it.

Reply:
> Also I copied from ROX_DISABLE_AUTOGENERATED_REGISTRIES :p

I would not speak myself free that I never introduced one 😆 

> I don't feel too strongly though so I can change it if you do feel strongly on it.

Please, I would appreciate that we try to avoid the negative inverted settings.

Reply:
@rukletsov My requested change would imply to change the default to `true` to not introduce breaking changes or limit functions in existing environments. Does this change your vote? 🤔

Reply:
@rukletsov My requested change would imply to change the default to `true` to not introduce breaking changes or limit functions in existing environments. Does this change your vote? 🤔

Reply:
Yea if the default is true that it's backwards compatible as a missing env var means it's included by default. The only change would be to set it to _false_ for managed services specifically

Reply:
As long as the default is "enable", I'm good : ).